salt/netapi/rest_cherrypy/app.py

```diff
@@ -8,6 +8,7 @@
 .. py:currentmodule:: salt.netapi.rest_cherrypy.app
 
 :depends:   - CherryPy Python module
+            - salt-api binary
```

Comment:
would "package" work better? i'm fine with executable but I think that 'binary' or 'package' would tip someone off to do a yum/apt install.